app/Connection.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use App\User;
+
+class Connection extends Model
+{
+    //Return a list of connections available to you
+    public static function getIncomingConnections(User $user)
+    {
+        $ids = self::where('user_id', $user->id)->pluck('owner_id')->toArray();
+        return array_map(function($id){ return User::find($id); }, $ids);
+    }
+
+    //Return a list of connections you've attempted to make
+    public static function getOutgoingConnections(User $user)
+    {
+        $ids = self::where('owner_id', $user->id)->pluck('user_id')->toArray();
+        return array_map(function($id){ return User::find($id); }, $ids);
+    }
+
+    public static function connect(User $from, User $to)
+    {
+        $connection = new Connection();
```

Comment:
Do these as one-liners:
```
return Connection::create([
'user_id' => 9001,
'owner_id' => 9004,
etc.
]);

Reply:
IMO it looks cleaner the other way but this is technically less lines. 💯